Assets/Mirror/Examples/Lobbey.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Why is the folder called "Lobbey"  as opposed to "Lobby"?   is there a name conflict with something else?

Reply:
Mystery bug - if you rename it the Lobby Scene field in LobbyManager changes to the offline scene or clears - I have no idea why.  Let me know if you figure it out.  I spent a couple hours trying.